donkeycarManagerFront/src/components/register.vue

```diff
@@ -81,31 +66,25 @@ export default {
     },
     async addUser () {
       this.playerTest = await srv.getPlayerByPseudo(this.pseudo)
-      console.log(this.playerTest)
-      console.log(this.playerTest.length)
       if (this.playerTest.length === 0){
-        console.log('pass')
         this.player = await srv.createPlayer(this.pseudo)
-        console.log('pass1.1')
         await srv.addJobs(this.player.player_id)
-        console.log('pass2')
-        if (this.waitingList.length < 2) {
-          this.attente = 'Maintenant'
-        } else if (this.waitingList.length >= 2 && this.waitingList.length < 4) {
-          this.attente = '15 minutes'
-        } else if (this.waitingList.length >= 4 && this.waitingList.length < 8) {
-          this.attente = '30 minutes'
-        } else if (this.waitingList.length >= 8 && this.waitingList.length < 12) {
-          this.attente = '45 minutes'
-        } else if (this.waitingList.length >= 12) {
-          this.attente = 'plus de 1 heure'
-        }
         this.numero = this.player.player_id
+      } else if (this.playerTest.length !== 0){
+        await srv.addJobs(this.playerTest[0].player_id)
```

Comment:
Duplicated, put it outside the conditional block will be cleaner, less lines => less bugs